Gemfile

```diff
@@ -65,6 +65,7 @@ group :development do
 
   # Speed up commands on slow machines / big apps [https://github.com/rails/spring]
   # gem "spring"
+  gem "letter_opener"
```

Comment:
great choice! `letter_opener` is a great option, but I know in some of the GoRails projects, they use `letter_opener_web`which gives a routes to view your delivered emails, kind of like the email previews in rails

https://github.com/fgrehm/letter_opener_web